spec/support/user_sign_in_helper.rb

```diff
@@ -8,4 +8,32 @@ def sign_in
 
     click_button 'Log in'
   end
+
+  def sign_in_with_adventures
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.